packages/squiggle-lang/src/public/SqProject/utils.ts

```diff
@@ -0,0 +1,169 @@
+// via https://github.com/6502/sha256/blob/main/sha256.js
```

Comment:
There are many SHA-256 implementations in various not-very-popular github projects and gists.

No idea which one is the best; some might be more performant. But we can optimize it later if necessary.

Reply:
Btw, git uses SHA-1, but SHA-1 is legacy. Not that we care too much about collision attacks, but still.

Reply:
(moved to source code comment)